front/src/modules/rollingStock/helpers/utils.ts

```diff
@@ -166,6 +156,48 @@ export const rollingStockEditorQueryArg = (
   base_power_class: data.basePowerClass || null,
 });
 
+export const isRollingStockFormValid = (
```

Comment:
I think the naming starting with "is" implies a boolean return. Would this naming be better ? (or something else)
```suggestion
export const RollingStockFormFieldsValidity = (
```

Reply:
Oh I didn't know that, yes I will rename the function :)